components/shared/components/Widget/components/panes/PaymentPane/PaymentPane.tsx

```diff
@@ -6,25 +6,20 @@ import {
   WidgetPane3BankRecurringProps,
   WidgetPane3BankSingleProps,
   WidgetPane3ReferralsProps,
-  WidgetPane3VippsRecurringProps,
-  WidgetPane3VippsSingleProps,
 } from "../../../types/WidgetProps";
-import { Pane } from "../Panes.style";
 import { ResultPane } from "./Bank/ResultPane";
 import { VippsPane } from "./Vipps/VippsPane";
+import { VippsProps } from "../../../types/VippsProps";
 
 export const PaymentPane: React.FC<{
-  text: WidgetPane3BankRecurringProps &
-    WidgetPane3BankSingleProps &
-    WidgetPane3VippsRecurringProps &
-    WidgetPane3VippsSingleProps &
-    WidgetPane3ReferralsProps;
-}> = ({ text }) => {
+  text: WidgetPane3BankRecurringProps & WidgetPane3BankSingleProps & WidgetPane3ReferralsProps;
+  vipps?: VippsProps;
+}> = ({ text, vipps }) => {
   const method = useSelector((state: State) => state.donation.method);
 
-  return (
-    <Pane>
```

Comment:
Removing the wrapping `<Pane />` element here, should be okay since `ResultPane` and `VippsPane` is already using it, right?

Reply:
Ooops, I didn't see this. It's weird if it's doubly nested, I'll check out the production code.

Reply:
Yes, it looks like it is indeed double nested! Yes, this is fine then :+1: